hriemann.cabal

```diff
@@ -15,7 +15,7 @@ cabal-version:       >=1.10
 
 library
   hs-source-dirs:      src
-  ghc-options:         -Wall
+  ghc-options:         -Wall -Wincomplete-uni-patterns -Wincomplete-record-updates -Wmissing-import-lists
```

Comment:
Doesn't `-Wall` get all the others?

Reply:
Apparently not: 
https://functor.tokyo/blog/2017-07-28-ghc-warnings-you-should-enable

Though according to an update on that post, apparently it's coming. Not sure when.

Reply:
Ah, AFAICT it's going to be in 8.6.